react/features/speaker-stats/components/TimeElapsed.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+
+import {
+    getHoursCount,
+    getMinutesCount,
+    getSecondsCount
+} from '../../base/util/timeUtils';
+
+/**
+ * React component for displaying total time elapsed. Converts a total count of
+ * milliseconds into a more humanized form: "# hours, # minutes, # seconds".
+ * With a time of 0, "0 seconds" will be returned.
+ *
+ * @extends Component
+ */
+class TimeElapsed extends Component {
```

Comment:
First I want to make clear that I would defer to your discretion as you have far more context and understanding of jitsi-meet. Moment has a lot of logic to it other than humanize and bringing it in for this task seemed like too much so taking on an extra dependency did not seem worthwhile just yet. Also, the logic needed to humanize milliseconds is not that complex (right now, anyway) so writing my own logic did not seem like a risk. If you foresee more time manipulation/interpretation/display being used in jitsi-meet, then I would definitely be in favor of bringing in moment in general.